mcs/class/Mono.Posix/Test/Mono.Unix.Native/RealTimeSignumTests.cs

```diff
@@ -23,21 +23,25 @@ namespace MonoTests.Mono.Unix.Native {
 	public class RealTimeSignumTest 
 	{
 		[Test]
-		[ExpectedException (typeof (ArgumentOutOfRangeException))]
 		public void TestRealTimeOutOfRange ()
 		{
-			if (!TestHelper.CanUseRealTimeSignals ())
-				return;
-			RealTimeSignum rts = new RealTimeSignum (int.MaxValue);
+			Assert.Throws<ArgumentOutOfRangeException> (() => {
+				if (!TestHelper.CanUseRealTimeSignals ())
+					return;
```

Comment:
please move this check outside the Assert.Throws here and in all the other tests.